_projects/image.md

```diff
@@ -9,4 +9,16 @@ docs: "https://docs.rs/image"
 docs_badge: "https://docs.rs/image/badge.svg"
 ---
 
-The `image` project.
+The `image` project is a comprehensive library and center stone of an ecosystem
+for decoding, encoding, representing, and manipulating images. The library
+originates spawned as part of the Piston project and soon enough some of its
+more popular encoders (such as `png`, `gif`, `tiff`) where split into separate
+crates. In 2019 the whole collection of image libraries was split into its own
+organization.
+
+The fractal in background of the header of this page was created with the image
+library! It is a variation of a long-standing
+[example](https://github.com/image-rs/image/blob/master/examples/fractal.rs)
+that can be found in the repository. The best part is that the crate is popular
+enough that you can expirement directly on [the Rust Playground
+play.rust-lang.org](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=2e8024c1a0fdcb9aebfab712e6f0a32d).
```

Comment:
This is nice and all, but afaik you can't really get any image output on the rust playground, so it's maybe not that interesting?

Reply:
True, it should really be a `wasm` in-browser playground that allows one to 'download' output images. But maybe trying out syntax is already some incentive?

Reply:
Yeah, just knowing that the crate can be used on the playground and for checking out the api is perfectly good enough reason to include the link.

Reply:
Maybe we can push the playground people towards implementing some sort of support for either downloading files or even better, viewing generated images directly in the playground?